ecs-init/docker/docker.go

```diff
@@ -427,6 +432,65 @@ func getDockerPluginDirBinds() []string {
 	return pluginBinds
 }
 
+// take pathPredicate as an argument for unit testing
+func getCapabilityExecBinds(pathPredicate func(path string, predicate func(fileInfo os.FileInfo) bool) (bool, error)) []string {
+	const (
+		capabilityExecName = "exec"
+
+		hostCapabilitiesResourcesRootDir      = "/home/ec2-user/capabilities-deps"
```

Comment:
nit: check lint

Reply:
do you mean `go fmt`? thought this is properly formatted already

Reply:
checked offline, resolving =)